roofit/roofitcore/inc/RooArgSet.h

```diff
@@ -23,12 +23,34 @@
 
 class RooArgList ;
 
+// # Original comment on USEMEMPOOLFORARGSET:
+//
 // Use a memory pool for RooArgSet.
 // RooFit assumes (e.g. for caching results) that arg sets that have the same pointer have
 // the same contents. Trying to remove that memory pool lead to wrong results, because the
 // OS *occasionally* returns the same address, and the caching goes wrong.
 // It's hard to track down, so disable this only when e.g. looking for memory leaks!
-#define USEMEMPOOLFORARGSET
+//
```

Comment:
Is this just a code comment or will be it  in the Doxygen documentation? 
As a comment it is fine, and it could be also included in the Release notes, 
It is fine instead to keep i

Reply:
It is only an inline comment for develop infos. For doxygen and the release notes, I have also written some paragraphs that don't go that much into detail.